ui-tests/tests/format_sql.test.ts

```diff
@@ -6,7 +6,7 @@ test('test format SQL', async ({ page }) => {
     await page.notebook.openByPath("sample.ipynb");
     await page.notebook.activate("sample.ipynb");
     await page.notebook.addCell("code", "%%sql\nselect * from table")
-    await page.getByTestId('format-btn').locator('button').click();
+    await page.getByTestId('format-btn').locator('button').click({ force: true });
```

Comment:
I faced this issue in the CI on the `main` branch as well as the current branch:
```
waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <span class="jp-ToolbarButtonComponent-label">Format SQL</span> intercepts pointer events
      - retrying click action, attempt #93 
  ```

Fixed using [this solution](https://github.com/microsoft/playwright/issues/13576#issuecomment-1129587745)